tests/js/spec/views/dashboardsV2/view.spec.tsx

```diff
@@ -0,0 +1,62 @@
+import {browserHistory} from 'react-router';
+
+import {initializeOrg} from 'sentry-test/initializeOrg';
+import {mountWithTheme} from 'sentry-test/reactTestingLibrary';
+
+import ViewEditDashboard from 'sentry/views/dashboardsV2/view';
+
+describe('Dashboards > ViewEditDashboard', function () {
+  const initialData = initializeOrg();
+
+  it('removes widget params from url and preserves selection params', async function () {
```

Comment:
We can drop the `async` here since we aren't using any `await`s, right?

Reply:
removed!